src/app/ui/layout/DocumentEditorLayout.module.css

```
@@ -0,0 +1,128 @@
+.container {
+  --document-editor-gap: var(--mantine-spacing-lg);
+  --document-editor-peek: 4rem;
+  --document-editor-preview-min-width: 19rem;
+  --document-editor-sidebar-min-width: 21rem;
+  --document-editor-sidebar-preferred-width: 27rem;
+  --document-editor-sticky-inset: 1rem;
+  --document-editor-fit-height: min(52rem, calc(100dvh - 2 * var(--document-editor-sticky-inset)));
+  container: document-editor-layout / inline-size;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)